internal/output/plaintext_fns.go

```diff
@@ -40,29 +40,32 @@ var ErrorPlaintextOutputFn = func(r resource) string {
 	}
 }
 
-// MultipleEmailPlaintextOutputFn converts the resource to plain text specifically for member data.
-var MultipleEmailPlaintextOutputFn = func(r resource) string {
-	return fmt.Sprintf("* %s (%s)", r["email"], r["_id"])
-}
-
-// MultipleIDPlaintextOutputFn converts the resource to plain text for data without a key.
-var MultipleIDPlaintextOutputFn = func(r resource) string {
-	return fmt.Sprintf("* %s (%s)", r["name"], r["_id"])
-}
-
-// MultiplePlaintextOutputFn converts the resource to plain text based on its name and key in a list.
+// MultiplePlaintextOutputFn converts the resource to plain text.
 var MultiplePlaintextOutputFn = func(r resource) string {
-	return fmt.Sprintf("* %s (%s)", r["name"], r["key"])
+	return fmt.Sprintf("* %s", SingularPlaintextOutputFn(r))
 }
 
 // SingularPlaintextOutputFn converts the resource to plain text based on its name and key.
 var SingularPlaintextOutputFn = func(r resource) string {
-	if r["name"] == nil {
-		return r["key"].(string)
-	}
-	if r["key"] == nil {
-		return r["name"].(string)
-	}
+	email := r["email"]
+	id := r["_id"]
+	key := r["key"]
+	name := r["name"]
 
-	return fmt.Sprintf("%s (%s)", r["name"], r["key"])
+	switch {
+	case name != nil && key != nil:
```

Comment:
This makes me think we should use the open api spec to tell us which fields to show in the output instead of all this logic. I'm not sure that would be simple to do.